src/test/java/org/datadog/jmxfetch/SimpleTestJavaAppMBean.java

```diff
@@ -45,4 +47,11 @@ public interface SimpleTestJavaAppMBean {
     TabularDataSupport getTabularDataSupport();
 
     CompositeData getNestedCompositeData();
+    Statistic getJeeCounter();
+    Statistic getJeeRange();
+    Statistic getJeeTime();
+    Statistic getJeeBoundary();
+    Statistic getJeeBoundedRange();
+    Statistic getJeeUnsupported();
+    Stats getJeeStat();
```

Comment:
minor: formatting doesn't match the rest of this interface

Reply:
added carriage returns